near/nep141-locker/src/lib.rs

```diff
@@ -100,6 +109,10 @@ pub trait ExtToken {
     ) -> Option<StorageBalance>;
 
     fn storage_balance_of(&mut self, account_id: Option<AccountId>) -> Option<StorageBalance>;
+
+    fn mint(&mut self, account_id: AccountId, amount: U128, msg: Option<String>);
+
+    fn burn(&mut self, amount: U128);
```

Comment:
aec1c070ef15048a9c5624c53b31bd933efaac25

Reply:
I want to split the interface here as well.

Reply:
I want to split the interface here as well.

Reply:
This will extend the code, because for now the part of the promise can be reusable 
`let transfer = ext_token::ext(token.clone()).with_attached_deposit(ONE_YOCTO);`, so it will not be possible to share this code like this.